test/server.test.js

```diff
@@ -10,7 +10,7 @@ var http = require('http');
 var stream = require('stream');
 
 var errors = require('restify-errors');
-var filed = require('filed');
+// var filed = require('filed');
```

Comment:
Why do we want to keep this commented out (as well as the test below)?

Reply:
I will get rid of it, this is not needed.